src/containers/Settings/constants.js

```diff
@@ -0,0 +1,30 @@
+export const DIFFICULTY_OPTIONS = [
```

Comment:
These should live in the public_modules bounties or bounty constants file.

Reply:
Wait - why are you using difficulty options?

Reply:
I copied and pasted another folder to start and forgot to remove 🙃